config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "environment":{
+    "production":{
+               "port":  3000
```

Comment:
After doing changes in any `.js Or . json ` please do indentation properly. Ctrl+shift+I